services/common/src/interfaces/reports/mineReportPageData.interface.ts

```diff
@@ -0,0 +1,9 @@
+import { IMineReport } from "../..";
+
+export interface IMineReportPageData {
```

Comment:
check out core-web ReportsTable for `IPageData<IMineReport>`! Generic type for the same thing, so we don't need to create as many interfaces